presto-hive/src/main/java/com/facebook/presto/hive/authentication/KerberosHadoopAuthentication.java

```diff
@@ -51,4 +45,22 @@ public UserGroupInformation getUserGroupInformation()
         Subject subject = kerberosAuthentication.getSubject();
         return createUserGroupInformationForSubject(subject);
     }
+
+    private Configuration validateHDFSConfiguration(HdfsConfiguration hdfsConfiguration)
+    {
+        Configuration config = hdfsConfiguration.getConfiguration(null, null);
```

Comment:
This is fragile. Add a zero parameters method to the `HdfsConfiguration` instead.